package.json

```diff
@@ -46,31 +46,31 @@
     "react-dom": ">=15"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
+    "@babel/cli": "^7.6.2",
+    "@babel/core": "^7.6.2",
+    "@babel/preset-env": "^7.6.2",
     "@babel/preset-flow": "^7",
     "@types/crypto-js": "^3",
     "@types/jest": "^23.3.14",
-    "@types/node": "^8.10.40",
-    "@types/react": "^16.8.5",
-    "@types/react-dom": "^16.8.2",
-    "ts-loader": "^5.3.3",
+    "@types/node": "^8.10.54",
+    "@types/react": "^16.9.3",
+    "@types/react-dom": "^16.9.1",
     "babel-plugin-inline-replace-variables": "^1",
-    "jest": "^24",
+    "jest": "^24.9.0",
     "jsdom": "^11",
-    "payment-icons": "^1",
-    "react": "^16.8.3",
-    "react-dom": "^16.8.3",
-    "react-test-renderer": "^16.8.3",
-    "ts-jest": "^24",
+    "payment-icons": "1.1.0",
```

Comment:
I explicitly pinned payment-icons.
Because we use the package as a test fixture in https://github.com/bitjourney/react-from-html/blob/6ed65533a874b5f64e15cdbd52fa8f268cf251e8/test/svg-snapshots.test.ts#L59
It must not be changed.